src/makecart.c

```diff
@@ -0,0 +1,671 @@
+#define _POSIX_C_SOURCE 200809L
+
+#include "../src/cartridge.h"
+#include "../src/files.h"
```

Comment:
Was this path specification intentional?

Reply:
Whoopsie. This file had started life in another directory and was later moved back into ./src.